src/components/Pill/Pill.enums.ts

```diff
@@ -0,0 +1,9 @@
+export const PillVariants = {
+  solid: 'solid',
+  outline: 'outline',
```

Comment:
```suggestion
  outline: 'outline',
  painkillers: 'painkillers'
```
This is a perfect opportunity for an easter egg 😂

Reply:
If you come up with proper styling I'm happy to do it :D